ntoskrnl/io/iomgr/deviface.c

```diff
@@ -28,6 +29,257 @@ IopGetDeviceObjectFromDeviceInstance(PUNICODE_STRING DeviceInstance);
 
 static PWCHAR BaseKeyString = L"\\Registry\\Machine\\System\\CurrentControlSet\\Control\\DeviceClasses\\";
 
+/**
+ * @brief IopBuildSymbolicLink
+ * 
```

Comment:
```suggestion
 * @brief
```

And similar suggestions for your other functions.

Reply:
Done. Same as for the documentation of all other functions.